packages/widgets/sso-login/src/components/LoginBox/LoginBox.js

```diff
@@ -1,51 +1,31 @@
 import PropTypes from 'prop-types'
-import React from 'react'
-import styled from 'styled-components'
+import {useEffect} from 'react'
+import {react} from 'tocco-util'
 
 import {openLoginWindow} from '../../utils/loginWindow'
 import ProviderButton from '../ProviderButton/ProviderButton'
+import {StyledButtonContainer} from './StyledComponents'
+
+const LoginBox = props => {
+  const {loadProviders, autoLogin, providers, loginEndpoint, loginCompleted} = props
+  loadProviders()
```

Comment:
I think this should be done within a `useEffect`: 
```
useEffect(() => loadProviders(), [])
```
Right now it will be invoked on every render. Before it was only invoked once when the component was created.